dwave/system/testing.py

```diff
@@ -12,94 +12,84 @@
 #    See the License for the specific language governing permissions and
 #    limitations under the License.
 
+import warnings
 import collections
-import unittest.mock as mock
-from uuid import uuid4
-
 import dimod
+import dwave.cloud.computation
+import unittest.mock as mock
 import dwave_networkx as dnx
+
+from uuid import uuid4
 from tabu import TabuSampler
-import dwave.cloud.computation
+
 
 try:
-    from neal import SimulatedAnnealingSampler
+    #Simplest and efficient choice returning low energy states:
+    from greedy import SteepestDescentSampler as SubstituteSampler
 except ImportError:
-    from dimod import SimulatedAnnealingSampler
-
+    from dimod import SimulatedAnnealingSampler as SubstituteSampler
 import concurrent.futures
 import numpy as np
 
-
-
-
-class MockDWaveSampler(dimod.Sampler, dimod.Structured):
+class MockDWaveSampler(dwave.system.samplers.DWaveSampler):
```

Comment:
I would not recommend subclassing the QPU sampler. I don't see mock sampler as a QPU subtype. See [LSP](https://en.wikipedia.org/wiki/Liskov_substitution_principle).